### ISSUE TITLE
Binding errors with implicitly referenced objects

### ISSUE THREAD

**Issue by bclothier** [WaynePhillipsEA/twinbasic#500]
**Describe the bug**
It seems that whenever there's a references to objects via `appobject` or something like that, the binding don't correctly resolve the members, even though it should have. 

**To Reproduce**
1. Create a project. Add Access object library and Access Database Engine object library (AKA DAO) as references.
2. Put in the following code in a module:
```
Public Module Test
    Public Sub x()
        Dim dbe As DAO.DBEngine
        Dim db As DAO.Database
        Set dbe = New DAO.DBEngine
        Set db = dbe.OpenDatabase("C:/foo/bar.accdb")
        
        Debug.Print db.QueryDefs(0).Name
        Debug.Print CurrentDb.QueryDefs(0).Name
    End Sub
End Module
```

3. Observe the error on the line with `CurrentDb`
![image](https://user-images.githubusercontent.com/2367644/142729814-29e828f9-ff99-4f16-bd07-39262a88a30c.png)

![image](https://user-images.githubusercontent.com/2367644/142729833-db8a7d5c-d3af-44d0-8be8-7a6e21af0b01.png)

One workaround is to explicitly add `Item` method:
```
Debug.Print CurrentDb.QueryDefs.Item(0).Name
```

**Expected behavior**
There should be no errors. It seems that twinBasic can correctly detect that the `CurrentDb()` is a member of the `Access.Application`, which also is an `appobject`, and that it returns a `Database`. However It fails to see that the `QueryDefs` is a member of the `DAO.Database` on the `CurrentDb. Compare:

![image](https://user-images.githubusercontent.com/2367644/142729990-bccf36ca-4572-4e49-9766-d03dc7e64e45.png)

to:

![image](https://user-images.githubusercontent.com/2367644/142730001-7f315160-c2b3-4432-8ed2-33978bb77758.png)

Somehow twinBasic is treating `CurrentDb` as if it was latebound when it shouldn't be. That then has the side effect of preventing it from treating the `QueryDefs(0)` as invoking a default member access. 

**Desktop (please complete the following information):**
 - OS: Windows 10
 - twinBASIC compiler version 0.13.1

**Comment by WaynePhillipsEA** [comment]
Hmm yes, weird that it works via normal variable access, just not from the predeclared id.  I'll get that fixed.

**Comment by WaynePhillipsEA** [comment]
Fixed in v0.13.4.  Thanks for reporting!
